README.md

```diff
@@ -495,6 +512,8 @@ If a `ClusterApplicationResourceMapping` defines `containers`, the reconciler **
 
 If a `ClusterApplicationResourceMapping` defines `env` and `volumeMounts`, the reconciler **MUST** first resolve a set of candidate locations in the application resource addressed by the `ServiceBinding` for all available containers and then filter that collection by the `ServiceBinding` `.spec.application.containers` filter before applying the appropriate modification.
 
+If a `ServiceBinding` specifies a `.spec.applications.containers` value, and the value contains an `Int`-based index, that index **MUST** be used to filter the first entry in the `.containers`, `.env`, and `.volumeMounts` lists and all entries in those lists are ineligible for mapping.  If a `ServiceBinding` specifies a `.spec.applications.containers` value, and the value contains an `string`-based index that index **MUST** be used to filter all entries in the `.containers`, `.env`, and `.volumeMounts` lists.
```

Comment:
filtering `.containers` makes sense, but I'm not sure how to apply the same filter to `.env` and `.volumeMounts` since the filtering is based on their being a list of containers.

Reply:
When an `env` and `volumeMounts` mapping is used, should we reject any binding that tries to filter by container, or just ignore the filter?

Reply:
Do'h of course.  Man that was sloppy.

>  should we reject any binding that tries to filter by container
I say yes, reject the ambiguity.